cmd/manager/main.go

```diff
@@ -152,9 +152,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Add the Metrics Service and Service Monitor
-	if err := winmetrics.Add(ctx, cfg, namespace); err != nil {
-		log.Error(err, "failed to add Metrics Service and Service Monitor")
+	// Validate will verify if cluster monitoring is enabled in the operator namespace
+	if err := winmetrics.Validate(ctx, cfg, namespace); err != nil {
+		log.Error(err, "failed to validate if Metrics Service and Service Monitor are created")
```

Comment:
How about `error validating metric setup`?

We should be generating an event here so that the user is aware..